app/views/recipe_foods/shopping_list.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1> Shopping list</h1>
+<div class='shopping_counters'>
+<h3> Amount of food items to buy: <%= @items_to_buy %> </h3>
+<h3> Total value: $ <%= @total %> </h3>
+</div>
+<table>
+  <tr>
+    <th>Food</th>
+    <th>Quantity</th>
+    <th>Value</th>     
+  </tr>
+  <% @recipefoods.each do |recipefood|  %>
+    <% if recipefood.quantity - recipefood.food.quantity > 0 %>
+    <tr>
+      <td><%= recipefood.food.name %></td>
+      <td><%= recipefood.quantity - recipefood.food.quantity%> <%= recipefood.food.measurement_unit %></td>
+      <td>$ <%= (recipefood.quantity - recipefood.food.quantity) * recipefood.food.price %></td>       
+    </tr>     
+  <% end %>   
```

Comment:
- [ ] As the previous reviewer suggested kindly make sure to make use of partial views to create your shopping list table